chatkit-core/src/main/kotlin/com/pusher/chatkit/state/Actions.kt

```diff
@@ -1,9 +1,14 @@
 package com.pusher.chatkit.state
 
 import com.pusher.chatkit.rooms.state.JoinedRoomInternalType
+import com.pusher.chatkit.users.state.UserInternalType
 
 internal sealed class Action
 
+internal data class CurrentUserReceived(
```

Comment:
shouldn't we already be creating actions following the [specs](https://github.com/pusher/chatkit-sdk-specifications/blob/master/redux/actions.md#subscription-event-actions)?

Reply:
We decided to go with the "old world" and then complete the whole refactor in one as the next task, so I don't see a problem with this for now. Using the new action type would mean changing a bunch of other stuff that can be refactored together.

Reply:
Fine, I just remembered that we both initially agreed that the law is in place already and any merge should adhere ;) If that got relaxed when you both caught up later, I'm cool with that too :)